i18n/en.json

```diff
@@ -1215,9 +1215,10 @@
   "admin.license.enterprise.restart": "Restart Server",
   "admin.license.enterprise.restarting": "Restarting",
   "admin.license.enterprise.upgrade": "Upgrade to Enterprise Edition",
-  "admin.license.enterprise.upgrade.acceptTermsFinal": " .Upgrading will download the binary and update your team edition.",
+  "admin.license.enterprise.upgrade.acceptTermsFinal": ". Upgrading will download the binary and update your team edition.",
   "admin.license.enterprise.upgrade.acceptTermsInitial": "By clicking **Upgrade**, I agree to the terms of the Mattermost ",
   "admin.license.enterprise.upgrade.eeLicenseLink": "Enterprise Edition License",
+  "admin.license.enterprise.upgrade.upgradeAndTrialTerms": ". Upgrading will download the binary and update your team edition.",
```

Comment:
This string isn't going to work well with localization - splitting them apart like this might look fine for english but it may not translate well to other languages

Reply:
ok! could you be please more specific about which part could potentially bring problems when translating to other languages?